static/docs/user-guide/contributing/core.md

```diff
@@ -250,6 +251,26 @@ $ export AZURE_STORAGE_CONNECTION_STRING="DefaultEndpointsProtocol=http;AccountN
 
 <details>
 
+### Click for Google Drive testing instructions
+
+❗Do not share Google Drive access token with anyone to avoid unauthorized usage
+of your Google Drive.
```

Comment:
Continuation of https://github.com/iterative/dvc.org/pull/833#pullrequestreview-327103665

But is it DVC's responsibility to alert about G Drive security? If not, maybe just a note (md quote starting with `>`) will suffice after all @shcheklein?

Regardless, please use this text:

> Please remember that Google Drive access tokens are personal credentials and should not be shared with anyone, otherwise risking unauthorized usage of the Google Drive.

Reply:
I think it's good to have this for GD - it not a common knowledge with all these tokens involved into GD.

Reply:
quotes with `>` are not visible. they defeat the purpose to some extent for notes like this. I'm fine with `>` though. I don't like when we mix `>` with some **Note!** or something -they look not very clean to me.

But tbh - I'm fine with all of these option. Jorge, I think we'll let you decide on this :)

Reply:
I agree notes with `>` are not visible and perhaps we shouldn't mix with bold "**Note!**". That's kind of contradictory! Opened #848 for this.

in this case though, while I like having the note, I do see it as a side-note that could be completely omitted and not something that DVC can really control. So for that reason and since we don't really have emojis ATM, I'm going to take it back to a `>` note...

Should we open an issue to apply some basic emoji symbols throughout with some consistency though? This could be a nice improvement to our docs.

Reply:
Addressed in d9ab97f8